packages/core/src/routes/experience/classes/verifications/verification-records-map.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @overview Since map in TS does not support strict key value type checking, we have to manually define the key value type.
```

Comment:
1. `@fileoverview`
2. "strict key value type checking" sounds inaccurate, since from the code i can only see "type mapping".

Reply:
updated